src/services/generateToken.ts

```diff
@@ -0,0 +1,51 @@
+const generateToken = async () => {
+
+    try {
+        // POST request using fetch with set headers
+        const urlencoded = new URLSearchParams();
+        urlencoded.append("grant_type", "client_credentials");
+        urlencoded.append("client_id", process.env.REACT_APP_SPOTIFY_CLIENT_ID || '' );
+        urlencoded.append("client_secret", process.env.REACT_APPP_SPOTIFY_CLIENT_SECRET || '');
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (typo):** There seems to be a typo in the environment variable name `REACT_APPP_SPOTIFY_CLIENT_SECRET`. It likely should be `REACT_APP_SPOTIFY_CLIENT_SECRET` (with two 'P's instead of three). This typo could prevent the application from correctly accessing the client secret from the environment variables.